superbench/cli/_commands.py

```diff
@@ -59,4 +61,18 @@ def load_arguments(self, command):
                 nargs='+',
                 help='Extra arguments to override config_file.'
             )
+        with ArgumentsContext(self, 'result') as ac:
+            ac.argument(
+                'raw_data_file', options_list=('--raw-data-file', '-d'), type=str, help='The path of raw data file.'
+            )
+            ac.argument('rule_file', options_list=('--rule-file', '-r'), type=str, help='The path of rule file.')
+            ac.argument(
+                'baseline_file', options_list=('--baseline-file', '-b'), type=str, help='The path of baseline file.'
+            )
+            ac.argument(
+                'output_dir',
+                type=str,
+                help='Path to output directory, outputs/{datetime} will be used if not specified.'
+            )
+            ac.argument('output_format', type=str, help='The format of output, excel or json.')
```

Comment:
same, and rename to "output_file_format"? there already exists output format

Reply:
ok